resources/osd/vision.mcm

```diff
@@ -0,0 +1,16385 @@
+MAX7456
```

Comment:
The font should also be added here to make it available in the configurator: https://github.com/frozenskys/betaflight-configurator/blob/7c3bcef7e8e2c795bfe67c6fe2cd7ec70f5066aa/src/js/tabs/osd.js#L909

Reply:
Adding an extra font in the UI causes the font selection buttons to overflow space available - I can re-jig the UI to use a couple of rows of buttons - or as a quick workaround can just change the screen name of the "Extra Large" font to "XL"...?

Reply:
![image](https://user-images.githubusercontent.com/28024/43486207-11b60648-950b-11e8-9672-3b9a0a71f793.png)

![image](https://user-images.githubusercontent.com/28024/43486108-d986de3c-950a-11e8-9fe4-087b2b1a5fd0.png)